examples/ceed/ex1-volume.c

```diff
@@ -303,7 +303,7 @@ int main(int argc, const char *argv[]) {
   return 0;
 }
 
-int GetCartesianMeshSize(int dim, int degree, int prob_size, int num_xyz[dim]) {
+int GetCartesianMeshSize(int dim, int degree, int prob_size, int num_xyz[3]) {
```

Comment:
```suggestion
int GetCartesianMeshSize(int dim, int degree, int prob_size, int num_xyz[]) {
```
Isn't it shorter when `dim < 3`?

Reply:
Ah, the real issue is line 59

Reply:
fixed